queries.sql

```diff
@@ -80,4 +80,28 @@ SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1
 GROUP BY species;
  species |        avg         
 ---------+--------------------
- pokemon | 3.0000000000000000
\ No newline at end of file
+ pokemon | 3.0000000000000000
+
+
+
+--  MULTIPLE TABLES DAY 3 
+-- What animals belong to Melody Pond?
+SELECT name, full_name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name, species.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
```

Comment:
- Please fix the result of this query when you fix the issues with assigning the species for the animals suggested above and do not forget to replace the screenshot in your PR 👍 👍